portals/devportal/src/main/webapp/source/src/app/components/Apis/Details/SourceDownload.jsx

```diff
@@ -121,6 +121,27 @@ function SourceDownload(props) {
                 }));
             });
     };
+
+    /**
+     * Downloads the async api specification of the api for the provided environment
+     *
+     * @param {string} apiId uuid of the API
+     * @param {string} environment name of the environment
+     */
+    const downloadAsync = (apiId, environment) => {
+        const promiseAsync = apiClient.getAsyncApiSpecificationByAPIIdAndEnvironment(apiId, environment);
+        promiseAsync
+            .then((done) => {
+                Utils.downloadFile(done);
+            })
+            .catch((error) => {
+                console.log(error);
```

Comment:
Better to use `console.error` instead of `console.log`

Reply:
Changed